content/posts/2021/reana-0.7.4.md

```diff
@@ -0,0 +1,134 @@
+---
+title: "REANA 0.7.4 is released"
+date: 2021-07-07T14:00:00+01:00
+tags:
+  - release
+  - client
+  - cluster
+---
+
+We are glad to announce the release of REANA 0.7.4, an update
+allowing REANA users to specify a memory limit for Kubernetes compute backend jobs.
```

Comment:
We are glad to announce the release of REANA 0.7.4, a minor update 
allowing users and cluster administrators to specify memory limits for workflow jobs running on the Kubernetes compute backend platform. The release also improves the REANA client functionality for workflow parameter validation and contains other minor improvements and bug fixes.